### ISSUE TITLE
How do we include the name of the state in an authority's name?

### ISSUE THREAD

**Issue by henare** [openaustralia/righttoknow#612]
In some cases we need to add the name of the state to the authority name, e.g. when the authority name is very common across jurisdictions like the Department of Education. How should we do this?

**Comment by henare** [comment]
To work out the answer to this question @equivalentideas and I did some research.

**tl;dr Put the state at the start of the name**, e.g. _NSW Department of Premier and Cabinet_.

## Different representations

First we found all the different combinations of how you could do this. Here are examples of all 4 ways we found...
- NSW Department of Premier and Cabinet
- Independent Pricing and Regulatory Tribunal of NSW
- Aboriginal Education Consultative Group Incorporated, NSW
- Aboriginal Affairs - NSW

## Positives and negatives

Then we listed the positives and negatives of each...
- All just as searchable as each other

NSW Department of Premier and Cabinet
- Sounds like it's part of the official title when it's not
- It's a commonly used way of talking about the agency
- When searching for a name that is common across each state then this makes it easy to find the right one you want
- Clumps together some alphabetical listing - makes alphabetical listing much less useful
- Very good for "Dear Foobar" in email template

Independent Pricing and Regulatory Tribunal of NSW
- Sounds like it's part of the official title when it's not
- Doesn't separate the subject and state as well
- Initially looks less clumsy but probably doesn't work well in bigger lists
- Fine for "Dear Foobar" in email template
- You're adding 2 words instead of 1

Aboriginal Education Consultative Group Incorporated, NSW
- Looks less clumsy
- Doesn't separate the subject and state as well
- Even worse for "Dear Foobar" in email template

Aboriginal Affairs - NSW
- Looks clumsy
- Doesn't read well
- Does sort well in lists
- Important aspect is front-loaded
- Bad for "Dear Foobar" in email template
- Not used in the wild

## Use cases

Then we thought about the different use cases and also went back and added positives and negatives above. We then ranked these by importance to requesters:
1. Making a request (searching on make request page)
2. "Dear Foobar" in email template
3. Google search
4. Trying to find authority in alphabetical list
5. In lists where there are authorities in other jurisdictions

## What's already in use?

Then we did a bit of research about what's already in use:
- The Government - they use any old form, quite different ones on the same page
- Wikipedia - they use any old form, quite different ones on the same page
- NSW or New South Wales searches on Right To Know (~5% of searches) - People put the state at the front and back of names of authorities they're looking for. Approximately 56% put it at the front of the name.
- Nowhere uses the "Aboriginal Affairs - NSW" form

## Conclusion

We ranked the positives and negatives section and came to the conclusion that **the _NSW Department of Premier and Cabinet_ form was the best**.

However in reviewing how this would look against a sample of different real-world names we discovered that it doesn't work when referring to a legal entity, e.g. _Central Coast Regional Development Corporation_. In this case it's actually best to just remove the state. There may be other examples where we do need to add the state but we'll cross that bridge when we come to it.

**Comment by equivalentideas** [comment]
@henare Should we be updating these in the spreadsheet as we go, or would it be better to do it at another step?

**Comment by henare** [comment]
@equivalentideas it's up to you how you do it for the state you're working on but it needs to have been completely done before we import that state. So I'd suggest doing it first and then making sure any additional ones you add follow the standard.
